test/cowsay_test.js

```diff
@@ -0,0 +1,9 @@
+const expect = require('chai').expect;
+const cowsay =require('../lib/cowsay.js');
+
+describe('cowsay function', function() {
+  it('This is a trip', function() {
+    const Expected = 'cowsay';
+    expect(cowsay('This is a trip')).to.eql(Expected);
```

Comment:
what can we expect once the cowsay function returns the inputted string concatenated with the string representing the cow